app/javascript/src/maps.js

```diff
@@ -152,15 +155,15 @@ class GPickerMap {
 class GMultiMap {
   constructor(canvas) {
     this.canvas = canvas[0];
-    this.init = canvas.data('init');
-    this.notices = canvas.data('notices');
+    this.init = canvas.data("init");
+    this.notices = canvas.data("notices");
   }
 
   show() {
     const map = initMap(
       this.canvas,
       [this.init.latitude, this.init.longitude],
-      this.init.zoom,
+      this.init.zoom
```

Comment:
i would like to keep trailing commas for ruby and js. less chance of merge conflicts.

Reply:
done. Please keep the [remarks](https://prettier.io/docs/en/options.html#trailing-commas) on "trailingComma: all" in mind.